<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc 2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>SmartCalc</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="5">
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="btn_eq">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6" colspan="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btn_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="btn_sum">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="btn_back">
          <property name="text">
           <string>&lt;-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="btn_x">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="decimals">
           <number>7</number>
          </property>
          <property name="minimum">
           <double>-1000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="btn_6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="btn_sub">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="btn_clr">
          <property name="text">
           <string>CLR</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>X FROM:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>-1000000</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>-5</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="btn_7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btn_8">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="btn_9">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="btn_mul">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="btn_sin">
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>X TO:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="minimum">
           <number>-1000000</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btn_dot">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btn_0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="btn_pow">
          <property name="text">
           <string>^</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="btn_div">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="btn_asin">
          <property name="text">
           <string>asin</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Y FROM:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="7">
         <widget class="QSpinBox" name="spinBox_3">
          <property name="minimum">
           <number>-1000000</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>-5</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="btn_bopen">
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="btn_bclose">
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="btn_sqrt">
          <property name="text">
           <string>sqrt</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="btn_mod">
          <property name="text">
           <string>mod</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="btn_cos">
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Y TO:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="7">
         <widget class="QSpinBox" name="spinBox_4">
          <property name="minimum">
           <number>-1000000</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="btn_ln">
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="btn_log">
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="btn_tan">
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="btn_atan">
          <property name="text">
           <string>atan</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="btn_acos">
          <property name="text">
           <string>acos</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5" colspan="3">
         <widget class="QPushButton" name="btn_chart">
          <property name="text">
           <string>Chart</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="8">
         <widget class="QCustomPlot" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Кредитный калькулятор</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Сумма</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>1000000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox_4">
          <item>
           <property name="text">
            <string>₽</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>$</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>€</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Срок </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_5">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>месяцев</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>лет</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Процентная ставка</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Тип</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QTableWidget" name="table_credit">
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Месяц</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Платеж</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Проценты</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Погашено</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Остаток</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QPushButton" name="btn_cred_calc">
          <property name="text">
           <string>Рассчитать</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>аннуитетный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>дифференцированный</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Депозитный калькулятор</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Сумма</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="text">
           <string>1000000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox_6">
          <item>
           <property name="text">
            <string>₽</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>$</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>€</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Срок </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_6">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBox_5">
          <item>
           <property name="text">
            <string>месяцев</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>лет</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Процентная ставка</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Налоговая ставка</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Капитализация процентов</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Периодичность выплат</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Пополнения</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="2">
            <widget class="QTableWidget" name="table_cap">
             <attribute name="verticalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Тип</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Период</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Дата</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Сумма</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Удалить</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Тип</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_9">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Пополнение</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Снятие</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Периодичность</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox_8">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Разовое</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Раз в месяц</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Раз в 2 месяца</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Раз в квартал</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Раз в полгода</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Раз в год</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Дата</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="dateEdit">
             <property name="calendarPopup">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Сумма</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="text">
              <string>10000</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="btn_add_cap">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QTableWidget" name="table_dep">
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Начислено процентов</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Вклад пополнен</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Остаток на вкладе</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_7">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Да</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Нет</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_2">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>Каждый день</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Каждую неделю</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Раз в месяц</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Раз в квартал</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Раз в полгода</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Раз в год</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>В конце срока</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QPushButton" name="btn_dep_calc">
          <property name="text">
           <string>Рассчитать</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">../qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
